raven/utils/serializer/base.py

```diff
@@ -174,7 +174,17 @@ class FloatSerializer(Serializer):
     types = (float,)
 
     def serialize(self, value, **kwargs):
-        return float(value)
+        value = float(value)
+
+        if value == float('inf'):
```

Comment:
Given this is a repr serializer, why don't we return what Python would actually render.

For example:


```python
>>> repr(float('-inf'))
'-inf'
```

Reply:
I tried to preserve the behavior that Sentry uses to normalize infinity/nan, but thinking about it there's no way this would actually break stuff. Will fix.